tests/integration/conftest.py

```diff
@@ -356,17 +366,37 @@ def project_based_user(client, rand_gen):
 @pytest.fixture
 def project_pack(client):
     projects = [
-        client.create_project(name=f"user-proj-{idx}") for idx in range(2)
+        client.create_project(name=f"user-proj-{idx}",
+                              queue_mode=QueueMode.Batch,
+                              media_type=MediaType.Image) for idx in range(2)
     ]
     yield projects
     for proj in projects:
         proj.delete()
 
 
 @pytest.fixture
-def configured_project(project, client, rand_gen, image_url):
-    dataset = client.create_dataset(name=rand_gen(str), projects=project)
-    dataset.create_data_row(row_data=image_url)
+def initial_dataset(client, rand_gen):
+    dataset = client.create_dataset(name=rand_gen(str))
+    yield dataset
+
+
+@pytest.fixture
+def configured_project(project, initial_dataset, client, rand_gen, image_url):
+    dataset = initial_dataset
+    data_row_id = dataset.create_data_row(row_data=image_url).uid
+
+    project._wait_until_data_rows_are_processed(
+        data_row_ids=[data_row_id],
+        wait_processing_max_seconds=DATA_ROW_PROCESSING_WAIT_TIMEOUT_SECONDS,
+        sleep_interval=DATA_ROW_PROCESSING_WAIT_SLEEP_INTERNAL_SECONDS)
```

Comment:
the below `create_batch` function calls `_wait_until_data_rows_are_processed` internally, so no need for this here